tests/unit/archive_base_test.py

```diff
@@ -235,7 +235,10 @@ async def test_get_hyperlinks(docname, pattern, links, request, html_docs):
 def test_validate_archive(datapackage, test_results, success):
```

Comment:
nit: probably we should change the name of this test if the method under test is changing.

Reply:
Fixed